CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.0.2]
+### Fixed
+- Bump `@metamask/eth-json-rpc-provider` from `^4.1.1` to `^4.1.5` ([#273](https://github.com/MetaMask/eth-block-tracker/pull/273))
```

Comment:
Since we are listing this under Fixed, perhaps we should also specify what issue was being fixed?

Reply:
I'm thinking in general it's just the bugfixes provided in that package, and interested users can refer to the changelog/release notes there? Unless there is something specific you have in mind for this package!

Reply:
Makes sense, but then maybe treating it as other generic package bumps under `Changed` would be better?

I don't have a strong opinion on this